packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcPreview.test.ts

```diff
@@ -357,7 +358,7 @@ describe('forceLightningLwcPreview', () => {
       }
     } as fs.Stats);
     showQuickPickStub.resolves(desktopQuickPick);
-    await forceLightningLwcPreview(mockLwcFilePathUri);
+    await lwcPreview(mockLwcFilePathUri);
```

Comment:
`forceLightningLwcPreview` now decides whether `lwcPreview` or `lwcPreviewContainerMode` is called. `isSFDXContainerMode()` returns `false` when running in the desktop. Assuming `isSFDXContainerMode()` to be false, updating the file to test `lwcPreview` function.